src/python/gem5/resources/resource.py

```diff
@@ -650,11 +650,15 @@ class SuiteResource(AbstractResource):
 
     def __init__(
         self,
-        workloads: Dict["WorkloadResource", Set[str]] = {},
+        workloads: Dict[Tuple[str, str], "WorkloadResource"] = {},
+        workloads_id_map: Dict[Tuple[str, str], Set[str]] = {},
```

Comment:
**MUST** update the documentation below. This new parameter is not obvious at all. Please update for all new parameters :)